libs/komencikit/src/verifyWallet.ts

```diff
@@ -26,9 +28,21 @@ export const verifyWallet = async (
   expectedSigner: Address
 ): Promise<Result<true, WalletValidationError>> => {
   const code = await contractKit.connection.web3.eth.getCode(metaTxWalletAddress)
+  const eip1167Match = EIP1167_BYTECODE_REGEXP.exec(code)
```

Comment:
In theory, we should expect this to be true always going forward, right? i.e. every wallet build with this version of komencikit should hit the v2 deploy wallet endpoint, and get an EIP-1167 proxy.

I wonder if we should have something in the `else` statement that we can alert on since IIUC we should never enter that branch.

Reply:
Not exactly we'll still have wallets in the wild which are of the direct kind we will for sure need to verify them on komenci's side when future actions will need to happen as meta transactions.

Reply:
Yeah I misunderstood how this function is used, I thought it was only used when valora verified the proxy, but as you note Komenci does this on every request.